requirements/tox/mypy_requirements.txt

```diff
@@ -0,0 +1 @@
+mypi
```

Comment:
can you please pin mypi

Reply:
Sure, kept it as is, as it's unused but I agree its a good idea to pin.